spec/game_spec.rb

```diff
@@ -0,0 +1,173 @@
+require 'game'
+require 'board'
+
+describe Game do
+
+    before do 
+        @board = Board.new
+        @game = Game.new(@board)
+    end
+
+    describe "#game_over?" do
+        context "given board.winning_row != nil" do
+            it "#game_over? returns true" do
```

Comment:
`#game_over?` is already used in `describe` no need to repeat it here.

Check how your codes look like by running it in documentation format:

`rspec --format d`